spark_parser/spark.py

```diff
@@ -160,7 +160,10 @@ def addRule(self, doc, func, _preprocess=True):
             else:
                 self.rules[lhs] = [ rule ]
             self.rule2func[rule] = fn
-            self.rule2name[rule] = func.__name__[2:]
+            if func.__name__ == '<lambda>': #STRANIKS_SCAN 29/02/2016
```

Comment:
Would it be that hard to just do what I ask and give a worked out example? I would greatly appreciate that over this kind of back and forth.